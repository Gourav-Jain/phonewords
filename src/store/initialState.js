export default {
  phoneword: {
    showLoader: false,
    error: false,
    listOfWords: [],
  },
};
