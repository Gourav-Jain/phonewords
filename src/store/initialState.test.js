import initialState from "./initialState";

describe("store", () => {
    test("initialState", () => {
        expect(initialState).toMatchSnapshot();
    });
});