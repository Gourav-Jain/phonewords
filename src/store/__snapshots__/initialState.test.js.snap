// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`store initialState 1`] = `
Object {
  "phoneword": Object {
    "error": false,
    "listOfWords": Array [],
    "showLoader": false,
  },
}
`;
